### DISCUSSION TITLE
Updating Projected Load Growth from AEO

### DISCUSSION THREAD

**Discussion by davidbadler** [NREL/ReEDS-2.0#151] [Q&A: Data] [answered]
Hello,

A question about the demand data in ReEDS. My understanding is the projected growth in electricity demand (by region) is in inputs\loaddata. We are working on updating these to reflect demand projections from EIA's AEO2023. For the load data using AEO (e.g. demand_AEO_2022_reference.csv), it appears as though the load in each region is normalized to 1 in 2010 and the relative growth in demand in subsequent years is relative to that 2010 value. Is that correct? If so, what data are you using for the historical values from say 2010-2021 (or the latest available data year) to match up with/normalize the projections of power sector demand by region from AEO?

Or perhaps more succinctly, what would be the best way to update these csvs to incorporate the latest demand growth projections from AEO2023? 

Thanks very much for your help.

**Comment by wesleyjcole** [comment, marked as answer]
The demand growth projections from AEO are slightly challenging to update because the behind the meter PV needs to be included in the load growth projections for ReEDS (because ReEDS explicitly models rooftop PV capacity and generation).  If you are interested in the specific process, it's probably best if we just chatted directly (you can connect at wesley.cole@nrel.gov).  Or if you just want the AEO 2023 demand files formatted for ReEDS, you can also send me a note and I can send you the input csv files (we've already put them into our internal version of the model).

Finally, the demand growth from AEO 2022 to AEO 2023 didn't change much in terms of overall electricity demand, so you are not likely to see much difference in switching to AEO 2023.

![image](https://github.com/NREL/ReEDS_OpenAccess/assets/49044852/dd18bd0d-bf1b-4d26-9010-e8c9baf5e04d)

**Reply by davidbadler** [reply]
Thanks - this is very helpful and good to know. I will follow up with you directly. Thank you!
